### ISSUE TITLE
Duplicate entry dex files

### ISSUE THREAD

**Issue by fxhereng** [wseemann/FFmpegMediaMetadataRetriever#87]
I'm trying to have FFmpegMediaPlayer and FFmpegMetadataRetriever working on the same project but I'm facing this issue : 

```
Error:Execution failed for task ':app:transformClassesWithJarMergingForDebug'.
> com.android.build.transform.api.TransformException: java.util.zip.ZipException: duplicate entry: wseemann/media/Metadata.class
```

I had a Dex error as well, I solved it putting:
```
    defaultConfig {
        .....
        multiDexEnabled true
        .....
 }
```

Thanks!

**Comment by ekzarov** [comment]
SAME Issue! Cannot have both in one project. I have Android Studio. Pls help to fix it! Thanks...

**Comment by wseemann** [comment]
This is fixed in the latest version (v1.0.5).
